### PR TITLE
Add Pyhton example

### PR DESCRIPTION
**Pull request by tvolge**

### CODE REVIEW

File: examples/pushData/pushData.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import databox
+from databox.rest import ApiException
```
Comment (alekseimi) [inline examples/pushData/pushData.py]: Unused import, although we probably should use it when handling exceptions.

File: examples/pushData/pushData.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from pprint import pprint
```
Comment (alekseimi) [inline examples/pushData/pushData.py]: Unused import.

File: examples/pushData/pushData.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    username = "<YOUR-CUSTOM-DATA-TOKEN>",
+    password = ""
+)
+
+with databox.ApiClient(configuration, "Accept", "application/vnd.databox.v2+json",) as api_client:    
+    api_instance = databox.DefaultApi(api_client)    
+    push_data = [{"key": "sales2", "value": 100, "unit": "USD", "date": "2021-01-01T00:00:00Z" }]
+    try:
+        api_instance.data_post(push_data=push_data)        
+    except Exception as e:
```
Comment (alekseimi) [inline examples/pushData/pushData.py]: I assume line 15 is raising an `ApiException` for actionable exceptions.

File: examples/pushData/pushData.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        print("Exception: %s\n" % e)
```
Comment (alekseimi) [inline examples/pushData/pushData.py]: By providing this example, we encourage the use of the ApiException.

```suggestion
    except ApiException as e:
        # Handle exceptions that occur during the API call, such as invalid data or authentication issues
        print("API Exception occurred: %s\n" % e)
    except Exception as e:
        # Handle any other unexpected exceptions
        print("An unexpected error occurred: %s\n" % e)
```

File: examples/pushData/pushData.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import databox
+from databox.rest import ApiException
+from pprint import pprint
+
+configuration = databox.Configuration(
+    host = "https://push.databox.com",
+    username = "<YOUR-CUSTOM-DATA-TOKEN>",
+    password = ""
+)
+
```
Comment (alekseimi) [inline examples/pushData/pushData.py]:
```suggestion
# Configuration setup for the Databox API client
# The API token is used as the username for authentication
# It's recommended to store your API token securely, e.g., in an environment variable
```

File: examples/pushData/pushData.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+with databox.ApiClient(configuration, "Accept", "application/vnd.databox.v2+json",) as api_client:    
+    api_instance = databox.DefaultApi(api_client)    
+    push_data = [{"key": "sales2", "value": 100, "unit": "USD", "date": "2021-01-01T00:00:00Z" }]
```
Comment (alekseimi) [inline examples/pushData/pushData.py]:
```suggestion
    # Prepare the data you want to push to Databox
    # The 'key' should match a metric in your Databox account, 'value' is the data point, 'unit' is optional, and 'date' is the timestamp of the data point
    push_data = [{"key": "sales2", "value": 100, "unit": "USD", "date": "2021-01-01T00:00:00Z" }]
```

**Review by alekseimi — CHANGES_REQUESTED** [review]
Apart from minor issues that detract from the readability of the code, looks good. 

Additionally: The file (module) name `pushData.py` is in camel case. The PEP-8 standard expects snake case, e.g `push_data.py`

Since this is an example, I took the liberty to add additional clarifying comments.

**Review by bwiz — APPROVED** [review]

**Review by alekseimi — APPROVED** [review]

**Pull request merged**